test/studies/shootout/meteor/kbrady/meteor-parallel-alt.chpl

```diff
@@ -358,8 +352,8 @@ proc searchLinearHelper(in board, in pos, in used, in placed,
 //
 record BackoffSpinLock {
   var l: atomic bool,
-      lockAttempts: uint,
-      maxLockAttempts = (2**16-1): uint;
+      lockAttempts = 0,
+      maxLockAttempts = (2**16-1);
```

Comment:
I forgot @ben-albrecht was out today.  Chatting with @ronawho by email, his concern was wraparound.  I convinced him that it wouldn't happen fast enough with a 64-bit int and he realized he'd been thinking of C 32-bit ints.  So I'm going to assume we're good here.